R/estimate_KM.R

```diff
@@ -1,9 +1,6 @@
-#' @title Wrapper for Kaplan Meier Time-to-Event analysis
+#' @title Wrapper for Kaplan-Meier Time-to-Event analysis
 #'
-#' @description This function is a wrapper around \code{survival::survfit.formula} to perform a Kaplan-Meier analysis, assuming right-censored data.
-#'    The function expects that the data has been filtered on the parameter (PARAM/PARAMCD) of interest. All NA values in the CNSR, AVAL and strata
```

Comment:
This should be kept. You can generalize it if you prefer, but it is important information for the programmer.

Reply:
I moved this text to a section in the help file

Reply:
I'll recheck later this week.

Reply:
One proposal is to mention CDISC compliant data model and explicitly ADTTE and how the function leverages it in the decription.  For example, we could add the following qualifer:

`The function is designed to leverage the \link[CDISC ADaM ADTTE data model](https://www.cdisc.org/standards/foundational/adam/adam-basic-data-structure-bds-time-event-tte-analyses-v1-0) and more explicity the conventions and controlled vocabulary of the data model. Functionality is also supplied to handle non ADTTE compiant data sets.  `